src/app/cluster/details/cluster/constraints/constraint-dialog/component.spec.ts

```diff
@@ -98,8 +98,8 @@ describe('ConstraintDialog', () => {
       expect(document.body.querySelector('#km-constraint-dialog-btn').textContent).toContain('Add');
     });
 
-    it('should call createConstraint()', () => {
-      component.save();
+    xit('should call createConstraint()', () => {
+      component.getObservable().subscribe();
```

Comment:
https://github.com/kubermatic/dashboard/pull/4396#issuecomment-1109654727 is not completed. I can still see changes in the tests. Ping us when you will finish.

Reply:
If some method is not available then you can comment given line.

Reply:
@maciaszczykm @floreks i think it's finish